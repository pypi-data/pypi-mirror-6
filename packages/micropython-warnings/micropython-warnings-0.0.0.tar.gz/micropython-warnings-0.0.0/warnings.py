# MicroPython doesn't support warnings
