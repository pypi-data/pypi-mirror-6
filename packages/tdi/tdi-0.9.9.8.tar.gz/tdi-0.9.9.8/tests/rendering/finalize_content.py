#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi import html
from tdi.model_adapters import PreRenderWrapper, RenderAdapter

template = html.from_string("""
<foo>
    <bar tdi:overlay=">-foo"/>
</foo>
""".lstrip()).overlay(html.from_string("""
<tdi tdi:overlay="-foo">
    <script tdi="test">JAVASCRIPT</script>
</tdi>
""".lstrip()))

class Model(object):
    def render_test(self, node):
        node.raw.content = node.raw.content.replace('SCR', ' hey ')

def adapter(model):
    return PreRenderWrapper(RenderAdapter(model))

model = Model()
html.from_string(template.render_string(None, adapter=adapter)).render(
    model, startnode="test"
)
print
