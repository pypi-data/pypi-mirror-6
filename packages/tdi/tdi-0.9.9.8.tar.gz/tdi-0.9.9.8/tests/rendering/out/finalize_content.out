<script>JAVA hey IPT</script>
